sdk/python/packages/flet-core/src/flet_core/local_authentication/mac_authentication.py

```diff
@@ -0,0 +1,55 @@
+class MacLocalAuth:
+    def __init__(self) -> None:
+        import ctypes
+        import sys
+        from LocalAuthentication import LAContext
+        from LocalAuthentication import (
+            LAPolicyDeviceOwnerAuthentication,
+        )
+
+        self.LAContext = LAContext
+        self.LAPolicyDeviceOwnerAuthentication = LAPolicyDeviceOwnerAuthentication
+
+        c = ctypes.cdll.LoadLibrary(None)
+        PY3 = sys.version_info[0] >= 3
+        if PY3:
+            self.DISPATCH_TIME_FOREVER = sys.maxsize
+        else:
+            self.DISPATCH_TIME_FOREVER = sys.maxint
```

Comment:
Do we need this?

I dont think `flet` is supported in python 2.

Reply:
Thanks, I missed that.

Will update it.